includes/admin/class-wc-admin-addons.php

```diff
@@ -39,6 +39,65 @@ public static function get_featured() {
 		}
 	}
 
+	/**
+	 * Build transient name
+	 *
+	 * @return string transient name
+	 */
+	public static function build_transient_name( $category, $term, $country ) {
```

Comment:
This function is a bit confusing, it takes 3 mandatory params, but it overrides the $transient_name var every time. So if you specify all 3 params then it will always just return the country transient.

Reply:
Maybe I don't understand the comment:  https://repl.it/repls/JollyWorrisomeBarracuda

It builds a string with all three if given all three.  I can add `= null` as defaults if that is better.